loader.js

```diff
@@ -815,6 +834,24 @@ var Module = null;
    EmscriptenRunner.prototype.requestFullScreen = function () {
    };
 
+   /*
+    * PC98DosBoxRunner
+    */
+  function PC98DosBoxRunner() {
+    return EmscriptenRunner.apply(this, arguments);
+  }
+  PC98DosBoxRunner.prototype = Object.create(EmscriptenRunner.prototype);
+  PC98DosBoxRunner.prototype.start = function () {
+    FS.symlink('/emulator/y/dosbox-default.conf', '/dosbox.conf');
```

Comment:
I almost missed this: it does make setup simpler, but it'll make things much more difficult in the long run. (IA would have to include the config file in every emulated piece of software, which is infeasible.) Instead, the user should be able to specify a url the same way they specify the url to the software itself. Of course, there should be a reasonable default for the case where the user doesn't specify a config file. Can we just leave it empty in that case?

Reply:
I added this line because the DOSBox-X need a dosbox.conf to run in PC-98 mode correctly, otherwise the emulator will crashes when it starts. But if user don't have a dosbox.conf in there game ZIP package, they also can load a dosbox.conf with the mountFile function. I will remove this line and update my example.